src/pages/Reader.tsx

```diff
@@ -56,11 +58,40 @@ class Reader extends React.Component<any, State> {
         if (parsedSettings.menu) {
           readerSettings.menu = parsedSettings.menu;
         }
+        if (parsedSettings.autoLoadChapter) {
+          readerSettings.autoLoadChapter = parsedSettings.autoLoadChapter;
+        }
       }
       catch (err) {
         console.log('No settings found.')
       }
     }
+    const keyBindings = {
+      previousPage: "K",
+      nextPage: "J",
+      previousChapter: "H",
+      nextChapter: "L",
+    }
```

Comment:
why shift+hjkl?
also for full page navigation `d` and `u` are more commonly used. `j` and `k` are just for small nudges.

and then for page nav i think `n` and `p` might be better?

Reply:
hjkl are fine too but i think it shouldn't be shift+hjkl

Reply:
Where are u getting shift from

Reply:
since its capital HJKL